packages/entities/entities-plugins/src/components/DeleteCustomPluginSchemaModal.vue

```diff
@@ -0,0 +1,205 @@
+    is-visible
+    :title="title"
+    @canceled="handleCancel"
+  >
+    <template #body-content>
+      <!-- TODO: -->
+      <!-- <div v-if="isPluginSchemaInUse">
+        <i18n-t
+          keypath="configuration.plugins.list.deleteCustomPlugin.pluginSchemaInUseText"
+          scope="global"
+          tag="p"
+        >
+          <template #name>
+            <strong>{{ plugin?.name }}</strong>
+          </template>
+        </i18n-t>
+      </div> -->
+
+      <!-- TODO: v-else -->
+      <div>
+        <div
+          v-if="errorMessage"
+          class="error-wrapper"
+          data-testid="error-message"
+        >
+          <KAlert
+            :alert-message="errorMessage"
+            appearance="danger"
+          />
+        </div>
+        <form
+          id="delete-custom-plugin-form"
+          @submit.prevent="handleSubmit"
+        >
+          <p>
+            {{ t('delete.confirmModalText1') }}
+            <strong>{{ plugin?.name }}</strong>?
+          </p>
+          <p>{{ t('delete.confirmModalText2') }}</p>
+
+          <!--   <EntityDeleteModal
+      :action-pending="isDeletePending"
+      :description="t('delete.description')"
+      :entity-name="pluginToBeDeleted && (pluginToBeDeleted.instance_name || pluginToBeDeleted.name || pluginToBeDeleted.id)"
+      :entity-type="EntityTypes.Plugin"
+      :error="deleteModalError"
+      :title="t('delete.title')"
+      :visible="isDeleteModalVisible"
+      @cancel="hideDeleteModal"
+      @proceed="confirmDelete"
+    /> -->
+          <!-- TODO: just use KPrompt??? -->
+          <!-- TODO: i18n-t element instead
+            <p class="confirm-text">
+            {{ helpText.confirm_1 }} "<strong>{{ plugin?.name }}</strong>" {{ helpText.confirm_2 }}.
+          </p> -->
+          <KInput
+            v-model.trim="customPluginNameFormValue"
+            data-testid="confirmation-input"
+            type="text"
+          />
+        </form>
+      </div>
+    </template>
+    <template #action-buttons>
+      <div>
+        <KButton
+          appearance="outline"
+          class="cancel-button"
+          data-testid="delete-custom-plugin-form-cancel"
+          :disabled="isLoading"
+          @click="handleCancel"
+        >
+          {{ t('actions.cancel') }}
+        </KButton>
+        <KButton
+          v-if="isPluginSchemaInUse"
+          appearance="primary"
+          data-testid="go-to-plugins-list"
+          :to="{
+            name: 'plugins'
+            /*  params: {
+              control_plane_id: config.app.controlPlaneId
+            } */
+          }"
+        >
+          {{ t('actions.go_to_plugins') }}
+        </KButton>
+        <KButton
+          v-else
+          appearance="danger"
+          data-testid="delete-custom-plugin-form-submit"
+          :disabled="isDeleteButtonDisabled"
+          form="delete-custom-plugin-form"
+          :icon="isLoading ? 'spinner' : undefined"
```

Comment:
suggestion: instead of using the `icon` prop that is going away, can you just slot in the `ProgressIcon` from `@kong/icons`? Same comment if you're using any other `icon` props in the PR.

Reply:
See above